### PR TITLE
Menu filter & switch themes should be on by default

### PR DESCRIPTION
**Pull request by hishamco**
Address the comment https://github.com/OrchardCMS/OrchardCore/issues/7420#issuecomment-718506915

### CODE REVIEW

**Comment by agriffard** [comment]
Will the default theme will be the light one?

**Comment by hishamco** [comment]
> Will the default theme will be the light one?

Yep, this just will show the button that enable you to switch between the themes

File: src/OrchardCore.Modules/OrchardCore.Admin/Admin/AdminSettings.cs [hunk 1]
```diff
@@ -2,8 +2,8 @@ namespace OrchardCore.Admin.Models
 {
     public class AdminSettings
     {
-        public bool DisplayMenuFilter { get; set; }
+        public bool DisplayMenuFilter { get; set; } = true;
 
-        public bool DisplayDarkMode { get; set; }
+        public bool DisplayDarkMode { get; set; } = true;
```
Comment (hishamco) [inline src/OrchardCore.Modules/OrchardCore.Admin/Admin/AdminSettings.cs:7]: @agriffard as the property said display the dark mode but not enable it, so we 're safe ;)

**Comment by agriffard** [comment]
Please wait that #7420 is fixed (admin menu filter) before merging this.

**Comment by Skrypt** [comment]
The default theme is defined by your OS current dark mode. If you are under Mac OS and you enabled dark mode on your OS it will show the dark admin theme by default ... If you click on the button then it stores your preference in the LocalStorage of your browser. So, once we enable the dark mode the default is not the light theme. If you don't enable it then the default is the light theme.

**Comment by hishamco** [comment]
> If you are under Mac OS for example and you enabled dark mode on your OS it will show the dark admin theme by default

How the theme is OS related?!!

**Comment by Skrypt** [comment]
Dude I showed you the line that does that and I added a comment on the PR. It's a new feature that has been added to browser that started by Safari on Mac OS. When they added the Mac OS X dark mode they also added an option to have a dark mode on websites to trigger automatically.

https://css-tricks.com/a-complete-guide-to-dark-mode-on-the-web/

Now Bootstrap doesn't support it yet but it's planned for version 6 isssh.

**Comment by hishamco** [comment]
Thanks, I didn't know about this feature, but can we enable the light theme instead of set the dark theme one by default, at least as @deanmarcussen suggested the user will notice there's a new feature. I agreed with him to show the feature but not enable it by default

**Comment by Skrypt** [comment]
And this is why I did not enable it by default 😄

**Comment by hishamco** [comment]
You 're the man ;)

What if disable the dark mode then and persist it `darkMode = false;` this should work?

**Comment by Skrypt** [comment]
When you disable the dark mode the javascript doesn't apply changes so even if the LocalStorage does hold the `darkmode = true` it won't apply it.

**Pull request merged**